### PR TITLE
Evo 1284 post search filter

### PR DESCRIPTION
**Pull request by thomasgwatson**
Post search available. Want to get this one reviewed and merged while I work on the other tidbits of this ticket.

#1284 has some screenshots

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -66,6 +66,7 @@
     "@draft-js-plugins/inline-toolbar": "^4.1.2",
     "@draft-js-plugins/linkify": "^4.2.0",
     "@draft-js-plugins/mention": "^5.1.1",
+    "@emoji-mart/react": "^1.0.1",
```
Comment (thomasgwatson) [inline package.json:69]: I backed these changes OUT.... 👎

File: src/store/models/Person.js [hunk 2]
```diff
@@ -40,8 +40,7 @@ Person.fields = {
     through: 'PersonSkillsToLearn',
     throughFields: [ 'person', 'skillToLearn' ]
   }),
-  postsTotal: attr(),
-  votesTotal: attr()
```
Comment (tibetsprague) [inline src/store/models/Person.js:44]: snuck in?

**Review by tibetsprague — APPROVED** [review]
LGTM

File: src/store/models/Person.js (hunk 2 above)
Comment (thomasgwatson) [reply src/store/models/Person.js:44]: It did... I swear I stashed all the changes... This attr isn't used anywhere anywhoo

**Pull request merged**